content/docs/api-reference/dvcfs.md

```diff
@@ -0,0 +1,66 @@
+# DvcFileSystem
+
+DvcFileSystem provides a pythonic file interface (aka
+[fsspec](https://filesystem-spec.readthedocs.io/)) for a DVC repo and provides a
+read-only filesystem-like operations like `ls`, `du`, `glob`, `get`, `download`,
+etc.
+
+DvcFileSystem provides a single view of all the files/directories in your
+repository, be it Git-tracked or DVC-tracked, or untracked (in case of a local
+repository). DvcFileSystem is smart to reuse the files in your cache directory
+(if present) and can otherwise stream/fetch the files from your default remote.
+
+### Basic Usage
+
+```py
+>>> from dvc.fs.dvc import _DvcFileSystem as DvcFileSystem
+# opening a local repository
+>>> fs = DvcFileSystem("/path/to/local/repository")
```

Comment:
Does this represent the current state of the repo? It looks like `_DvcFileSystem.__init__()` expects a `Repo` instance, and it doesn't seem to work for me with a `str` arg:

```python
>>> fs = DvcFileSystem(".")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/dave/miniforge3/envs/dvc/lib/python3.10/site-packages/fsspec/spec.py", line 76, in __call__
    obj = super().__call__(*args, **kwargs)
  File "/Users/dave/Code/dvc/dvc/fs/dvc.py", line 118, in __init__
    self.hash_jobs = repo.fs.hash_jobs
AttributeError: 'str' object has no attribute 'fs'
```

Reply:
See https://github.com/iterative/dvc.org/pull/3932#issuecomment-1242232523.